README.md

```diff
@@ -37,8 +37,9 @@ module.exports = {
   reporters: ['dot', 'junit'],
   reporterOptions: {
     outputDir: './',
-    outputFileFormat: function(opts) { // optional
-        return `results-${opts.cid}.${opts.capabilities}.xml`
+    outputFileFormat: function(opts) {
+      var suiteName = opts.config.suite.replace(/([^a-z0-9]+)/gi, '-');
```

Comment:
Do you think it is better to make opts have all the configs, or just the suite name is enough? 

Suite name is what I need for myslef. I'm not sure if other users would want additional information from configs such as framework, etc, which they should be using at their own discretion.